project1/styles.css

```diff
@@ -0,0 +1,60 @@
+h1 {
+    font-size: 38px;
+    color: black;
+    font-style: italic;
+    text-align: center;
+    text-transform: uppercase;
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+}
+
+.box {
+    background-color: rgb(183, 190, 117);
+    border: 10px solid rgb(183, 190, 117);
+    height: 125px;
+    width: 300px;
+    padding: 30px;
+    margin: auto;
+}
+
+h2 {
+    font-size: 26px;
+    font-style: italic;
+    text-align: center;
+    text-transform: uppercase;
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+    background-color: whitesmoke;
+}
+
+blockquote {
+    font-size: 26px;
+    background-color: whitesmoke;
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+    text-align: center;
+}
+
+p {
+    font-size: 18px;
+    background-color: whitesmoke;
+    line-height: 1.5;
+    text-align: center;
```

Comment:
Recall the class convo about centering body text. In general, it's okay in small doses, but for multiple paragraphs, I recommend either left or justified alignment (depending on how wide the text is).